lib/storage/client.js

```diff
@@ -1044,6 +1047,65 @@ class Client extends CryptoConsumer {
     const params = { id_string: noteName }
     return shared.issueNoteChallenge(this, params, meta)
   }
+
+  /**
+   * Make a request object for the create group request
+   * @param {string} name
+   * @param {Array} capabilities
+   * @param {KeyPair} encryptionKeyPair
+   *
+   * @return {Group}
+   */
+  async createGroupRequest(name, capabilities = [], encryptionKeyPair) {
+    const groupKeyPair = await this.crypto.generateKeypair()
+    const eak = await this.crypto.encryptPrivateKey(
```

Comment:
nit: same as above, let's call this pk or or similar to indicate it is a private key, not an eak